### ISSUE TITLE
Inference examples not working on docker container. (pycbc/pycbc-el7:latest)

### ISSUE THREAD

**Issue by kevinkvothe** [ligo-cbc/pycbc#1648]
I've been trying to run the examples described in the inference documentation (https://ligo-cbc.github.io/pycbc/latest/html/inference.html) and I've had many errors that made me impossilble to get the outputs.
The only thing that works right is the injection creation (with lalapps_inspinj).
After creating the inference.ini by copy-pasting and executing pucbc_inference with the kombine sampler, I get:
**File "/home/pycbc/pycbc-software/lib/python2.7/site-packages/PyCBC-785ddb-py2.7.egg/pycbc/inference/sampler_kombine.py", line 85, in __init__
            processes=pool.count)
            TypeError: __init__() got an unexpected keyword argument 'transd'**
I solved it by removing the extra keyword arguments, so the 83-84-85 line stays as 
"sampler = kombine.Sampler(nwalkers, ndim, likelihood_call, pool, transd)".
Then, I execute pycbc_inference again and, after five mins, I get:
            **File "/home/pycbc/pycbc-software/lib/python2.7/site-packages/PyCBC-6a5f32-py2.7.egg/pycbc/inference/sampler_kombine.py", line 144, in run
            res = self._sampler.run_mcmc(niterations, p0=p0, **kwargs)
            AttributeError: 'Sampler' object has no attribute 'run_mcmc'**
Which I don't know how to solve. I don't surely know if I made some mistake or the container base is wrong, but I made everything by copy-pasting from the documentation.

On the other part, I try making the inference with real data, and I get (after changing the priors as described in the documentation):
**File "/home/pycbc/pycbc-software/lib/python2.7/site-packages/PyCBC-6a5f32-py2.7.egg/pycbc/frame.py", line 238, in datafind_connection raise ValueError(err)
        ValueError: Trying to obtain the ligo datafind server url from the environment, ${LIGO_DATAFIND_SERVER}, but that variable is not populated.**      

If somebody could explain what I'm doing wrong or if there's something to change, It would be very appreciated. Thanks.

**Comment by cdcapano** [comment]
@kevinkvothe The transd and `run_mcmc` issue your are seeing is due to a bug in kombine's version history. When the author of kombine first uploaded it to pypi, he made the version number 1.0. He later changed the version number to 0.5; the current most latest version is 0.8. The problem is, when you run setup.py, setup tools thinks that the 1.0 version is more recent than the 0.8 version, and so installs that, even though it is several years old. You can fix this by doing the following. In your virtual environment, do:
```
pip uninstall kombine
pip install kombine
```
This should uninstall the old version of kombine you have, then install the new one. (Pip is apparently smart enough to know that version 0.8 is more recent than 1.0.)

I thought kombine had fixed this issue, but apparently not. I'll put in a pull request on the pycbc side to look for the specific version of kombine until they do, which will fix this issue.

For the second part, you cannot use the real data in the manner described in the docs page if you are not a LIGO member. To run on real data, download the *gwf files from the LIGO open science center website for the even you are interested in running on:
https://losc.ligo.org
Then replace the `--frame-type` argument to `pycbc_inference` with `--frame-files H1:${H1_FILENAME} L1:${L1_FILENAME}`, where `${H1_FILENAME}` and `${L1_FILENAME}` are the locations of the gwf files you downloaded for H1 and L1, respectively. We'll add a note explaining this to the docs page.

**Comment by cmbiwer** [comment]
Hi, ``kommbine`` versions are kind of messed up on pypi.

Try:
```
pip install kombine==0.8.0
```

Right now I'm assuming you have version 1.1 which is actually an older version of ``kombine``.

We should probably change our ``requirements.txt`` and ``setup.py`` to explicitly choose 0.8.0 untill Ben pushes a new version.

**Comment by kevinkvothe** [comment]
Perfect, now it's working, I'll try the inference with real data using gwf files too. Thanks.

**Comment by cdcapano** [comment]
Fixed by #1649 and #1650.
